vm/kvm-unit-tests/runtest.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+. /usr/bin/rhts_environment.sh
+
+export ACCEL=kvm
+unset ARCH
+unset STANDALONE
+cpus=$(grep -c ^processor /proc/cpuinfo)
+
+# test can only run on hardware that supports virtualization 
+if egrep -q '(vmx|svm)' /proc/cpuinfo; then
+    echo "Hardware supports virtualization, proceeding" | tee -a $OUTPUTFILE
+else
+    echo "Skipping test, CPU doesn't support virtualization" | tee -a $OUTPUTFILE
+    rhts-report-result $TEST SKIP $OUTPUTFILE
+    exit
+fi
+
+# test should only run on a system with 1 or more cpus
+if [ "$cpus" > 1 ]; then
+    echo "You have sufficient CPU's to run the test" | tee -a $OUTPUTFILE
+else
+    echo "Skipping test, system requires > 1 CPU" | tee -a $OUTPUTFILE
+    rhts-report-result $TEST SKIP $OUTPUTFILE
+    exit
+fi
+
+# test is only supported on x86_64
+if [ "$(uname -i)" == "x86_64" ]; then
+    echo "Running on x86_64 arch" | tee -a $OUTPUTFILE
+else
+    echo "Skipping test, test is only supported on x86_64" | tee -a $OUTPUTFILE
+    rhts-report-result $TEST SKIP $OUTPUTFILE
+    exit
+fi
+
+# set the virt kernel parameters
+echo -e "options kvm force_emulation_prefix=1\noptions kvm enable_vmware_backdoor=1" > /etc/modprobe.d/kvm-ci.conf
+
+# reload the modules
+rmmod kvm_intel kvm_amd kvm
+modprobe kvm_intel kvm_amd kvm
+
+KVM_SYSFS=/sys/module/kvm/parameters/
+KVM_OPTIONS="enable_vmware_backdoor force_emulation_prefix"
+
+for opt in $KVM_OPTIONS; do
+  if [ ! -f "$KVM_SYSFS/$opt" ]; then
+    echo "kernel option $opt not set" | tee -a $OUTPUTFILE
```

Comment:
I only noticed this now, but should there be any result reporting before exiting here?

Reply:
@veruu yeah probably :-) I updated in both places